src/xdocs-test/resources/com/puppycrawl/tools/checkstyle/checks/naming/abstractclassname/XDocExample4.java

```diff
@@ -0,0 +1,17 @@
+/*
+AbstractClassName
+format = ^Generator.+$
+ignoreModifier = (default)false
+ignoreName = (default)false
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.naming.abstractclassname;
+
+class XDocExample4 {
+    abstract class GeneratorFirstClass {} // OK
+    abstract class SecondClass {} // violation '.* must match pattern '\^Generator\.\+\$''
+    class GeneratorThirdClass {} // violation '.* must be declared as 'abstract''
```

Comment:
XDocExample files should be identical by java code (wihtout comments).
Only comment are changing between Example1 to Example2.

Reply:
Done.